python/sparkdl/graph/utils.py

```diff
@@ -71,8 +72,7 @@ def get_op(tfobj_or_name, graph):
         raise TypeError('invalid op request for [type {}] {}'.format(type(name), name))
     _op_name = op_name(name, graph=None)
     op = graph.get_operation_by_name(_op_name)
-    assert op is not None, \
-        'cannot locate op {} in current graph'.format(_op_name)
+    assert isinstance(op, tf.Operation), 'expect tf.Operation, but got {}'.format(type(op))
```

Comment:
I think the previous error message was more clear. the current one sounds a little bit like you expected the user to give us the operation object, not our internal invocation of `get_operation_by_name`.